pants.toml

```diff
@@ -2,12 +2,18 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 [GLOBAL]
-pants_version = "2.9.0rc0"
+pants_version = "2.9.0rc1"
 backend_packages = [
-  "pants.backend.experimental.scala",
-  "pants.backend.experimental.scala.lint.scalafmt",
+  # This repository demonstrates a mix of Java and Scala, and so both backends are enabled. But each
+  # backend can be used independently, so there is no need to expose Scala BUILD file
+  # symbols if you have a pure-Java repository.
```

Comment:
Ditto pure-scala

Reply:
It's far less common to have a Scala repo with no Java at all, so I think it's fine not to belabor this point.